### ISSUE TITLE
loader.loadSound not accepting Filename file names

### ISSUE THREAD

**Issue by wezu** [panda3d/panda3d#458]
Trying to load a sound (sfx or music) when the name of the file is given as a Filename will cause a TypeError because the underling AudioManager.getSound() function expects a string. Other loader.load_* functions have no problems with Filename file names.
```
from panda3d.core import Filename
from direct.showbase import ShowBase
base=ShowBase.ShowBase()
wav=loader.load_sfx(Filename('audio/sfx/GUI_click.wav'))
```

**Comment by rdb** [comment]
I think this was addressed by 17bf50f1e8d3f136bea14cfdb30346d4aba9c974, could you test with the latest version?

**Comment by wezu** [comment]
Ah, ok. Works after updating.
